python/Ganga/Core/GangaRepository/SessionLock.py

```diff
@@ -274,16 +274,15 @@ def global_disk_lock(f):
     @functools.wraps(f)
     def decorated_global(self, *args, **kwds):
         with global_disk_lock.global_lock:
-            i=0
-            while i< 10:
-                i+=1
+            for i in range(10):
```

Comment:
Actually, this should be `range(11)` or the check below should be `if i == 9`. My mistake.